corehq/apps/fixtures/static/fixtures/js/lookup-manage.js

```diff
@@ -1,6 +1,22 @@
-var somethingWentWrong = $("#FailText").text();
-$(function () {
+/**
+ *  Handles fixtures' "Manage Tables" page.
+ */
+define([
+    "jquery",
+    "knockout",
+    "hq.helpers",
```

Comment:
Figuring out which modules to include can be a bit of a headache, especially for modules that aren't explicitly referenced in this file (e.g., `hq.helpers` is needed here because it sets up `koApplyBindings` as a jQuery plugin). Might want to set up some kind of "core" module with the major dependencies.